tools/tpm2_import.c

```diff
@@ -564,17 +583,18 @@ static bool on_option(char key, char *value) {
 bool tpm2_tool_onstart(tpm2_options **opts) {
 
     const struct option topts[] = {
-      { "input-key-file",     required_argument, NULL, 'k'},
-      { "parent-key-handle",  required_argument, NULL, 'H'},
-      { "parent-key-public",  required_argument, NULL, 'f'},
-      { "import-key-private", required_argument, NULL, 'r'},
-      { "import-key-public",  required_argument, NULL, 'q'},
+      { "input-key-file",     required_argument, NULL, 'k' },
+      { "parent-key-handle",  required_argument, NULL, 'H' },
+      { "parent-key-public",  required_argument, NULL, 'f' },
+      { "import-key-private", required_argument, NULL, 'r' },
+      { "import-key-public",  required_argument, NULL, 'q' },
```

Comment:
Please don't change the these entries, if you want to do some style cleanups do it as a separate commit.